teamscale-client/src/main/kotlin/com/teamscale/client/HttpUtils.kt

```diff
@@ -139,7 +139,9 @@ object HttpUtils {
 	 * Sets sensible defaults for the [okhttp3.OkHttpClient].
 	 */
 	private fun Builder.setTimeouts(readTimeout: Duration, writeTimeout: Duration) {
-		connectTimeout(Duration.ofSeconds(60))
+		// In a K8s environment, it might take quite some time until the K8s networking is fully initialized and a connection can be established.
+		// Just setting the connectTimout does only retry once, but not wait for the given duration if the ConnectException is thrown very quickly.
+		addInterceptor(ConnectExceptionRetryInterceptor(Duration.ofMinutes(1)))
```

Comment:
The Jira ticket mentions 5 minutes??

Reply:
Ah yeah, we discussed this on Slack that 5 mins are way too long. If we make it actually wait for one minute, which it did not before (see comment), we should be good (suggested by Fabi).